### DISCUSSION TITLE
"no-scrollbar" a custom class specifically to hide scrollbars on different browsers.

### DISCUSSION THREAD

**Discussion by MrItzreal** [tailwindlabs/tailwindcss#14093] [Ideas]
I started my coding journey in February 1st of this year and immediately fell in love with Tailwind CSS. I can't imagine ever using regular CSS for my projects again. While implementing a scrollbar feature for thumbnail images, I used `overflow-x-auto snap-x snap-mandatory whitespace-nowrap`, but discovered Tailwind doesn't have a built-in class to hide the scrollbar. Although it might be insignificant I was able to research and found an useful custom class that I wanted to share with you as a thank you for creating Tailwind. My apologies if this has already been submitted.

```
/* Custom utility to hide scrollbar on multiple Browsers*/
@layer utilities {
  /* Hide scrollbar for Chrome, Safari, and Opera */
  .no-scrollbar::-webkit-scrollbar {
    width: 0;
    height: 0;
  }

  /* Hide scrollbar for IE, Edge, and Firefox */
  .no-scrollbar {
    scrollbar-width: none;
  }
}
```

**Comment by lukewarlow** [comment]
https://github.com/tailwindlabs/tailwindcss/pull/14183 - adds this in the form of `scrollbar-none`
